client/reader/following/intro.jsx

```diff
@@ -3,22 +3,20 @@
  */
 import React from 'react';
 import { localize } from 'i18n-calypso';
-import { connect } from 'react-redux';
```

Comment:
No - we're going to use Calypso preferences instead, probably adding a new action called `deletePreference` which performs the same function as `requestGraduate`.

I'm happy to remove the comment lines in the meantime though 👍